lib/components/normal-components/PowerSource.ts

```diff
@@ -43,6 +43,14 @@ export class PowerSource extends NormalComponent<
       are_pins_interchangeable: false,
     } as any)
     this.source_component_id = source_component.source_component_id
+
+    if (props.voltage) {
+      db.simulation_voltage_source.insert({
+        voltage: props.voltage,
```

Comment:
we should also put a source_component_id on the simulation_voltage_source (I think it can be required)